### ISSUE TITLE
New opensource rolling restart utility

### ISSUE THREAD

**Issue by mvgorbunov** [ydb-platform/ydb-rolling-restart#3]
We need an opensource utility for rolling-restart cluster (storage, dynnodes) in the next environments:
1. Baremetal/VM 
2. Docker/k8s
3. Yandex Cloud specific environment (grpc Bastion for storage and Instance Group VM)

As a part of ydbops utility, using public  YDB API and package for CMS requests https://github.com/ydb-platform/ydb-rolling-restart

**Key features:**
- TBD

**Additional features:**
- TBD

**Out of scope:**
- Complicated tenants restart (e.g. "restart 10 databases in parallel but not more than 10% of each database nodes")

**Comment by shmel1k** [comment]
Hello! I have a proposal for new rolling restart: filter by datacenter option.

By now, we do not have such option and, if we want to have datacenter-by-datacenter update, we have to use "hosts-filter" option, where we set this hosts in advance(and pray that this list is constant and never changes)

Option "dc-filter"(or some more pleasant name) will allow to restart hosts in any provided DC, comma-separated

For example,
rolling --dc-filter=MAN,SAS

Or something like that

**Comment by Jorres** [comment]
Looks like this could be taken from `Ydb_Discovery.NodeLocation` in `Ydb_Maintenance.Node`. I will need to investigate whether this struct can be empty for some reason (just the gut feeling that it can...).

**Comment by Jorres** [comment]
--dc-filter not implemented yet, moved it to a separate issue in ydbops repository: https://github.com/ydb-platform/ydbops/issues/5
